### PR TITLE
THC 858 create search page layout

### PR DESCRIPTION
**Pull request by bzavhorodskyi**

### CODE REVIEW

File: src/components/pages/search/PlatformFilter/PlatformFilterBadge.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    return `primary`;
+  };
+
+  return (
+    <div
+      className="font-body"
+      style={{ display: "flex", alignItems: "center" }}
+    >
+      <span className="text-base font-medium">Platform</span>
+      <div className="search_platform-filter_badge text-sm">
```
Comment (Sporiff) [inline src/components/pages/search/PlatformFilter/PlatformFilterBadge.tsx:25]: This is another place where we could probably style this inline, just to cut down on arbitrary CSS.

```suggestion
      <div className="text-sm [&_span]:font-body [&_span]:text-sm">
```

**Review by Sporiff — APPROVED** [review]
@bzavhorodskyi This looks great to me 🎉  I've just taken a look if there's anywhere we could use inline styles rather than CSS files, but these are a suggestion only. I don't think it makes a difference either way.

File: src/components/Cards/SearchResultCard.astro [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  <div class="flex gap-x-5">
+    <PageBreadcrumbs breadcrumbs={breadcrumbs} withMargin={false} />
+  </div>
+  <a href={link} class="search_result-link flex flex-col">
+    <h5 class="text-heading-5 font-semibold text-search-primary mt-3 mb-4">
+      {title}
+    </h5>
+    <p class="text-heading-5" text-search-primary>{description}</p>
+  </a>
+</div>
```
[the remark this answers is not in the document]
Comment (bzavhorodskyi) [reply src/components/Cards/SearchResultCard.astro]: Thanks, fixed

File: src/components/Cards/search-result-card.css
[the remark this answers is not in the document]
Comment (bzavhorodskyi) [reply src/components/Cards/search-result-card.css:1]: Removed

File: src/components/pages/search/PlatformFilter/PlatformFilterBadge.tsx (hunk 1 above)
Comment (bzavhorodskyi) [reply src/components/pages/search/PlatformFilter/PlatformFilterBadge.tsx]: Fixed

**Pull request merged**